marlowe-runtime/.gitignore

```diff
@@ -0,0 +1 @@
+client_session_key.aes
```

Comment:
3 Comments:

1. Please don't add per-directory .gitignores (I know we have them elsewhere, but this is a legacy thing and should be cleaned up and moved to the root .gitignore)
2. What was this for?
3. For keeping local key files in repo, we already have a `treasury/` entry in our root .gitignore. If you need to keep a key somewhere in the repo hierarchy, make a `treasury` folder in the repo root and put your private stuff in there.

Reply:
No problems! It seems to be generated by yesod. I could investigate it a little deeper to understand what it is.

Reply:
Fixed it with the advice given in [this stackoverflow thread](https://stackoverflow.com/questions/43395944/prevent-yesod-from-generating-client-session-key-aes-file).